mteb/tasks/Classification/kor/KlueTC.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class KlueTC(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="KLUE-TC",
+        dataset={
+            "path": "klue",
```

Comment:
I think the path should be: `klue-ynat` as itis on HF: https://huggingface.co/datasets/yangwang825/klue-ynat. The name attribute is for selecting a subset.
```suggestion
            "path": "klue-ynat",
```

Reply:
This repo also has data from KLUE's YNAT subset, but it's probably better to refer to KLUE's official dataset path, which is maintained by the KLUE researchers at https://huggingface.co/datasets/klue/viewer/ynat.

Reply:
Ah my bad I wasn't on the right URL